go/vt/mysqlctl/replication.go

```diff
@@ -258,8 +258,9 @@ func (mysqld *Mysqld) SetReadOnly(on bool) error {
 	case true:
 		newState = "ReadOnly"
 	}
-	log.Infof("SetReadOnly setting connection setting of %s:%d to : %s",
-		mysqld.dbcfgs.Host, mysqld.dbcfgs.Port, newState)
+	params, _ := mysqld.dbcfgs.DbaWithDB().MysqlParams()
+	log.Infof("SetReadOnly setting connection setting of %s:%d, socket:%s to : %s",
+		params.Host, params.Port, params.UnixSocket, newState)
```

Comment:
We should probably check for errors here though as `params` might end up being `nil` leading to a panic on line 262

Reply:
I don't think it's useful to print the socket file path. From a debugging perspective we want to know which tablet alias this is operating on, not the socket file.
So my proposal is:
- only log this if host/port are set, meaning external tablet mode
- ensure that the caller logs this using tablet alias. that is information that is available higher up in the call stack, but not in this function.

Reply:
`setting connection setting` is also misleading. ReadOnly is being set globally not on the connection.

Reply:
After discussing with @deepthi offline, we agreed that since this is a vttablet log, the host port/socket that it is connected to, is not really interesting, because we only have 1 MySQL instance it is going to be connected to, so it going to run this query on that only. I have changed the log message and removed the host and port from it entirely.